env_configurations.go

```diff
@@ -18,15 +19,26 @@ func (ec EnvConfigurations) basePath(stackId, envId int64) string {
 	return fmt.Sprintf("/orgs/%s/stacks/%d/envs/%d/configuration", ec.Client.Config.OrgName, stackId, envId)
 }
 
-func (ec EnvConfigurations) Create(stackId, envId int64, file io.Reader) ([]types.Workspace, error) {
+func (ec EnvConfigurations) Create(stackId, envId int64, config, overrides string) ([]types.Workspace, error) {
 	body := &bytes.Buffer{}
 	writer := multipart.NewWriter(body)
 
-	part, err := writer.CreateFormFile("configuration", "previews.yml")
-	if err != nil {
-		return nil, err
+	if config != "" {
+		part, err := writer.CreateFormFile("config", "config.yml")
+		if err != nil {
+			return nil, err
+		}
+		configReader := strings.NewReader(config)
+		_, err = io.Copy(part, configReader)
```

Comment:
It's highly unlikely that this won't error, but if it does, the failure could be confusing.
It would be best to return an error here.

Reply:
Good catch, I missed that. I have a focus to provide good error messages right now too.